pyproject.toml

```diff
@@ -4,10 +4,13 @@ version = "0.1.0"
 description = ""
 authors = ["Francesco Pogliano <54809768+Cyangray@users.noreply.github.com>"]
 readme = "README.md"
+package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.12"
```

Comment:
I'd fix python version to 3.12. It's also sometimes good to take maybe second-to-latest python version for the project because there is always some lag, i.e. many packages may not be yet adapted to the latest python. But maybe this is not important at the moment and for the tools you're currently using.

Reply:
@Cyangray a hint, you can fix the python version like this: `">=3.12,<3.13"`